hudi-utilities/src/test/java/org/apache/hudi/utilities/functional/TestHoodieDeltaStreamer.java

```diff
@@ -449,6 +451,14 @@ static void assertAtleastNDeltaCommits(int minExpected, String tablePath, FileSy
       assertTrue(minExpected <= numDeltaCommits, "Got=" + numDeltaCommits + ", exp >=" + minExpected);
     }
 
+    static void assertAtLeastNCompletedReplaceCommits(int minExpected, String tablePath, DistributedFileSystem fs) {
```

Comment:
There is already `assertAtLeastNReplaceCommits` method. I think we can reuse that as it also fetches the completedReplaceTimeline.

Reply:
Sure, changed